### ISSUE TITLE
"Get-MgServiceAnnouncementMessage -All" only loads first page

### ISSUE THREAD

**Issue by buckbanzai** [microsoftgraph/msgraph-sdk-powershell#1141]
Hello,
When running the following command, it only outputs 100 records, where I would expect it to page through all available results (259 in this case).
```
$ServiceAnnouncementMessages = Get-MgServiceAnnouncementMessage -All -CountVariable 'ServiceAnnouncementMessageCount'
PS /Users/liam.steckler> $ServiceAnnouncementMessages.Count
100
PS /Users/liam.steckler> $ServiceAnnouncementMessageCount
259
```

SDK version 1.9.3, running on PowerShell Core 7.2.1 on macOS

**Comment by timayabi2020** [comment]
> Hello, When running the following command, it only outputs 100 records, where I would expect it to page through all available results (259 in this case).
> 
> ```
> $ServiceAnnouncementMessages = Get-MgServiceAnnouncementMessage -All -CountVariable 'ServiceAnnouncementMessageCount'
> PS /Users/liam.steckler> $ServiceAnnouncementMessages.Count
> 100
> PS /Users/liam.steckler> $ServiceAnnouncementMessageCount
> 259
> ```
> 
> SDK version 1.9.3, running on PowerShell Core 7.2.1 on macOS
@buckbanzai This is both a service and an SDK issue.  Whenever you provide the argument "-All" the SDK sends a request with a default page size of 100 (https://graph.microsoft.com/v1.0/admin/serviceAnnouncement/messages?$top=100), expecting next link to be provided ( "@odata.nextLink": "https://graph.microsoft.com/v1.0/admin/serviceAnnouncement/messages?$Select=id&$skip=100"). Unfortunately this is not happening, therefore a service issue has been raised to address this problem. 
Meanwhile on the sdk side we will remove the default page size so that we just work with the size we are getting from the service.
Below is a screenshot that summarizes the above explanation.
**Next link provided without specifying a page size**
```Get-MgServiceAnnouncementMessage -CountVariable 'ServiceAnnouncementMessageCount' -Debug```
![Screenshot 2022-04-05 172941](https://user-images.githubusercontent.com/10947120/161793633-275ce329-c52a-4b9b-82ce-978ef864e2cd.png)
**Page size specified yet there is no next link**
```Get-MgServiceAnnouncementMessage -All -CountVariable 'ServiceAnnouncementMessageCount' -Debug```
![Screenshot 2022-04-05 173153](https://user-images.githubusercontent.com/10947120/161793809-8ce062ad-520c-45f3-bfbe-da53c9e4d7b7.png)

**Comment by peombwa** [comment]
Will be fixed by #1361.

Closing as duplicate of #1351.
